drivers/usb/device/usb_dc_mcux.c

```diff
@@ -729,20 +778,20 @@ static void handle_transfer_msg(usb_device_callback_message_struct_t *cb_msg)
 	} else {
 		/* IN TOKEN */
 		if (USB_EP_DIR_IS_IN(ep)) {
-			if ((dev_data.address != 0) && (ep_abs_idx == 1)) {
+			if ((dev_data.usb_dev_struct.deviceAddress != 0) && (ep_abs_idx == 1)) {
```

Comment:
Please move this block to handle_transfer_in() function, NACK for a code block like this that goes over 80 characters.

Reply:
I am not clear on this comment. This line is setting the same field inside the SDK data structure, the field name has changed with the update to the latest SDK. This passes the compliance check.